Frontend/src/ui/App.jsx

```diff
@@ -11,8 +14,11 @@ export default function App({ children }) {
     setUserState(user)
   }, [])
   return (
-    <AuthContext.Provider value={{ user, setUser }}>
-      {children}
-    </AuthContext.Provider>
+    <QueryClientProvider client={queryClient}>
+      <AuthContext.Provider value={{ user, setUser }}>
+        {children}
+      </AuthContext.Provider>
+      <ReactQueryDevtools initialIsOpen={false} />
```

Comment:
What does this do?

Reply:
It is supposed to be some handy dev tools, https://tanstack.com/query/v4/docs/react/devtools, but I haven't had any use for them yet...